src/lib/registration/widgets/addons_selector.rb

```diff
@@ -0,0 +1,241 @@
+        super
+        # FIXME: only if item get selected?
+        label_event_handler(item)
+      end
+
+      # (see CWM::MultiStatusSelector#label_event_handler)
+      def label_event_handler(item)
+        details_widget.value = item.description
+      end
+
+      # Whether the filter must be shown
+      #
+      # @return [Boolean] true if there are any registered or released item; false otherwise
+      def show_filter?
+        @show_filter ||= @items.reject(&:registered_or_released?).any?
+      end
+
+      # Widget to display the filter, when needed
+      #
+      # @return [Filter, Yast::Term] Filter if the filter should be displayed; Empty otherwise
+      def filter_widget
+        @filter_widget ||= show_filter? ? Filter.new(self.class.release_only) : Empty()
+      end
+
+      # Widget to display the details area
+      #
+      # @return [CWM::RichText]
+      def details_widget
+        @details_widget ||= CWM::RichText.new
+      end
+
+      # Convenience widget to build the filter
+      class Filter < CWM::CheckBox
+        # Constructor
+        #
+        # @param value [Boolean] the initial value
+        def initialize(value)
+          @initial = value
+        end
+
+        # @macro seeAbstractWidget
+        def init
+          self.value = @initial
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("&Hide Development Versions")
+        end
+
+        # @macro seeAbstractWidget
+        def opt
+          [:notify]
+        end
+      end
+
+      # Internal class to represent an {Registration::Addon} as selectable item
+      #
+      # @note The addon status is directly managed by {Registration::Addon}. See delegated methods.
+      class Item < Item
+        extend Forwardable
+        include Yast::I18n
+
+        # Constructor
+        #
+        # @param addon [Registration::Addon] the addon to be represented
+        def initialize(addon)
+          textdomain "registration"
+
+          @addon = addon
+        end
+
+        # @!method toggle
+        #   Toggles the item
+        #   @note the logic to toggle an addon status is actually delegated to {Registration::Addon}
+        def_delegator :@addon, :toggle_selected, :toggle
+
+        # @!method enabled?
+        #   @return [Boolean] true if the addon is available; false when not.
+        def_delegator :@addon, :available?, :enabled?
+
+        # @!method description
+        #   @return [String] the item description, using richtext
+        def_delegators :@addon, :description
+
+        def id
+          @id ||= "#{addon.identifier}-#{addon.version}-#{addon.arch}"
+        end
+
+        # Builds and return the item label based on the addon label and availability
+        #
+        # @return [String] the item label
+        def label
+          @label ||=
+            if addon.available?
+              addon.label
+            else
+              _("%s (not available)") % addon.label
```

Comment:
Add `# TRANSLATORS: label for a not available module/extension, %s is replaced by the module/extension name` comment